website/docs/basics/namespaces.md

```diff
@@ -0,0 +1,35 @@
+---
+id: namespaces
+title: Namespaces
+sidebar_label: Namespaces
+---
+
+[![docs-source](https://img.shields.io/badge/source-eigthshift--libs-blue?style=for-the-badge&logo=php&labelColor=2a2a2a)](https://github.com/infinum/eightshift-libs)
+
+> PHP part of this project lives in `EightshiftLibs` namespace.
+
+But because WordPress lives in an global namespace we had to provide the way for your project to be unique. That is why we implemented [**Imposter**](https://github.com/infinum/imposter-plugin) script inside `composer.json`. Imposter adds Namespace prefix on all the packages inside Vendor folder that uses namespacing.
```

Comment:
```suggestion
But because WordPress lives in a global namespace, we had to provide the way for your project to be unique. That is why we implemented [**Imposter**](https://github.com/infinum/imposter-plugin) script inside `composer.json`. Imposter adds a namespace prefix on all the packages inside the `vendor` folder that uses namespacing.
```